app/services/builders/external_pull_request.rb

```diff
@@ -0,0 +1,18 @@
+module Builders
+  class ExternalPullRequest < BaseService
+    def initialize(pull_request_data, external_project)
+      @pull_request_data = pull_request_data
+      @external_project = external_project
+    end
+
+    def call
+      ::ExternalPullRequest.find_or_create_by!(github_id: @pull_request_data[:id]) do |pull_request|
```

Comment:
it was kind of the intention, at the end of the execution we need to create it or doing nothing, since we don't even track the status of these pull requests i didn't want to be doing DB executions, i think the most important attribute here is pull_request's url, so that's why i didn't put to much effort keeping track of the pull request.. do you think we should update the rest information of the pr anyway?

Reply:
Not at all! If that was the intention, I'm perfectly fine with it. It makes perfect sense! 👍